src/patternfly/components/Masthead/masthead.scss

```diff
@@ -50,6 +51,7 @@ $pf-v6-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "
   --#{$masthead}--m-display-inline__main--GridColumn: 1; 
   --#{$masthead}--m-display-inline__content--GridColumn: 2;
   --#{$masthead}--m-display-inline__content--Order: 0;
+  --#{$masthead}--m-display-stack__main--toggle--content--GridColumn: 2;
```

Comment:
Is this meant to be duplicated from (new) line 39, or is it meant to be `--#{$masthead}--m-display-inline__main--toggle--content--GridColumn` (which is used on line 94, but isn't defined anywhere)

Reply:
Doh! Should be `--m-display-inline`.